src/enclave/Enclave/ExpressionEvaluation.h

```diff
@@ -702,6 +702,92 @@ class FlatbuffersExpressionEvaluator {
       }
     }
 
+    case tuix::ExprUnion_In:
+    {
+      auto c = static_cast<const tuix::In *>(expr->expr());
+      size_t num_children = c->children()->size(); 
+      bool result = false;
+
+      if (num_children < 2){
+        throw std::runtime_error(std::string("In can't operate with fewer than 2 args, currently we have")
+          + std::to_string(num_children)); 
+      }
+
+      auto left_offset =  eval_helper(row, (*c->children())[0]);
+      const tuix::Field *left = flatbuffers::GetTemporaryPointer(builder, left_offset);
```

Comment:
We need to be able to handle `NULL` values, an [example](https://github.com/mc2-project/opaque/blob/master/src/enclave/Enclave/ExpressionEvaluation.h#L40) here.

Reply:
fixed